crates/engine/tree/src/tree/mod.rs

```diff
@@ -2730,7 +2729,10 @@ where
     ) -> Result<(), InsertBlockErrorKind> {
         // Get the builder once, outside the thread
         let Some(state_provider_builder) = self.state_provider_builder(block.parent_hash())? else {
-            trace!(target: "engine::tree", parent=%block.parent_hash(), "Could not get state provider builder for prewarm");
+            trace!(target: "engine::tree",
```

Comment:
can we do it in other places too, otherwise it looks weird imo
```suggestion
            trace!(
               target: "engine::tree",
```